src/pages/home/sidebar/OptionRow.js

```diff
@@ -129,12 +129,7 @@ const OptionRow = ({
                                         avatarImageURLs={option.icons}
                                         optionIsFocused={optionIsFocused}
                                         size={mode === 'compact' ? 'small' : 'default'}
-                                        styles={hovered && !optionIsFocused && {
-                                            secondAvatar: {
-                                                backgroundColor: themeColors.sidebarHover,
-                                                borderColor: themeColors.sidebarHover,
-                                            },
-                                        }}
+                                        secondAvatarStyle={hovered && !optionIsFocused && styles.secondAvatarHovered}
```

Comment:
if the short circuit returns `false` then we are returning `Boolean` and not `Object`

Reply:
Ok. Will empty `{}` is fine instead of `false`? `false will have no effect while parsed by react-native.

Reply:
Yep or `undefined` since we have a default prop I think? Up to you.

Reply:
p.s. it's causing a propTypes warning which is why I'm mentioning it.